diff --git a/configure.ac b/configure.ac
index 2c33b695795a..622aeb9079a6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -164,9 +164,6 @@ AC_CHECK_FUNCS(fexecve, [], [])
 AC_CHECK_FUNCS([gettid])
 AC_CHECK_FUNCS([mallinfo2])
 
-AC_CHECK_HEADER(uthash.h, , [AC_MSG_ERROR(
-["Couldn't find uthash.h...uthash-devel is missing"] )])
-
 
 echo .
 echo Checking for required libraries
diff --git a/src/Makefile.am b/src/Makefile.am
index 05d0f8e9ddd6..65a218992152 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -5,6 +5,9 @@ AM_CPPFLAGS = \
 	-I${top_srcdir} \
 	-I${top_srcdir}/src/library
 
+AM_CPPFLAGS += \
+	-I${top_srcdir}/uthash-2.3.0/include
+
 sbin_PROGRAMS = fapolicyd fapolicyd-cli
 lib_LTLIBRARIES= libfapolicyd.la
 
diff --git a/src/library/rpm-backend.c b/src/library/rpm-backend.c
index 392fd9176b0c..e93e13369096 100644
--- a/src/library/rpm-backend.c
+++ b/src/library/rpm-backend.c
@@ -46,7 +46,7 @@
 #include <fnmatch.h>
 #include <sys/mman.h>
 
-#include <uthash.h>
+#include "uthash.h"
 
 #include "message.h"
 #include "gcc-attributes.h"
